### DISCUSSION TITLE
sjtusetup里title和title*之间多了一行空行

### DISCUSSION THREAD

**Discussion by shiwanghua** [sjtug/SJTUThesis#808] [提问] [answered]
就报了四个错，说括号不匹配，找了好久，还总是掉线，一段一段替换才找出来。

![image](https://user-images.githubusercontent.com/29914854/169503025-2110bc11-9942-421b-aa3d-88c344eac729.png)

内容太分散了，分成了好多个tex，点击右边的内容，无法直接定位到tex里的那一行或者直接定位错了文件，这就很不友好。

《\sjtuthesis\ 使用文档》在哪里？

**Comment by LogCreative** [comment]
主文档是 `main.tex`，使用说明主要见 [README](https://github.com/sjtug/SJTUThesis/blob/master/README.md)、编译出来的文档内容，文档类选项见 [开发文档](https://github.com/sjtug/SJTUTeX/releases/) 的第一个Assets。

**Reply by LogCreative** [reply]
分成多个文件是为了便于内容管理，不需要在一个大文件内定位内容。如果遇到 SyncTeX 定位错误，你需要检查文档是否正常编译。

**Reply by shiwanghua** [reply]
比如我要ctrl+f 全文查找某个东西，就得一个个文件打开去搜索，或者统计某个单词出现了多少次

**Reply by LogCreative** [reply]
可以考虑 [本地编译](https://github.com/sjtug/SJTUThesis/wiki/%E6%9C%AC%E5%9C%B0%E4%BD%BF%E7%94%A8%E8%AF%B4%E6%98%8E)，比如 VS Code 编辑器提供了全局搜索功能。

**Reply by shiwanghua** [reply]
没法多人同时编辑，不知道能不能看历史

**Reply by LogCreative** [reply]
那可以考虑将子文件内容手动拷贝到主文件中进行管理。本地编译如果需要多人编辑、查看历史可以考虑使用版本管理系统比如 [Git](https://git-scm.com/) 与代码托管平台 [GitHub](https://github.com) 等。

**Comment by skyzh** [comment, marked as answer]
如果不想分到多个 tex 文件，直接改 main.tex 就行了。
